tensorflow/core/kernels/spectrogram_test_utils.cc

```diff
@@ -68,7 +68,29 @@ bool ReadRawFloatFileToComplexVector(
   std::vector<std::complex<double> > data_row;
   int row_counter = 0;
   int offset = 0;
+  char arr[4];
   const int end = data_string.size();
+#if __BYTE_ORDER__ == __ORDER_BIG_ENDIAN__
```

Comment:
Could you move the big endian code below to only modify the parts that changed instead of copying this whole block?

Reply:
@jhseu, Thank you for your feedback. That would involve check for endianness inside [while loop](https://github.com/linux-on-ibm-z/tensorflow/blob/bbdd966581370316ee9bd5e2a2c7430e2e60ef9c/tensorflow/core/kernels/spectrogram_test_utils.cc#L94). To avoid this unnecessary check at every iteration, I moved it above the while loop(Also, this ensures LE code remains untouched). Do let me know, if you still feel I need to make the change.

Reply:
It's an #if preprocessing directive, so it should matter if it's inside the while loop, right?

Reply:
@jhseu,  Thank you for the correction. Apologies, I missed it. I have moved the Big endian code below. Could you please have a look?